lib/connector.js

```diff
@@ -218,6 +218,45 @@ Connector.prototype.id = function(model, prop) {
   return p && p.id;
 };
 
+/**
+ * Return the database mapping name of the property of the model if it exists.
+ * Otherwise return the property name.
+ * @param {String} model The target model name
+ * @param {String} prop The property name
+ */
+Connector.prototype.propertyMapping = function(model, property) {
```

Comment:
Are we going to remove the `SQLConnector.prototype.column` function?

If so I'd prefer to keep the name same as `Connector.prototype.column`. So that existing tests & code won't break. The new behaviour will be: 
  - all connectors have `column` function
  - (if a specific connector has its own implementation, it will override this one, which means nothing gonna change)

And I think you should delete the `SQLConnector.prototype.column` and `SQLConnector.prototype.idColumn` in the same PR.

WDYT?